Packs/Tenable_io/ReleaseNotes/2_3_8.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Tenable Vulnerability Management (formerly Tenable.io)
+- Updated the Docker image to: *demisto/python3:3.12.12.5490952*.
+
+- Updated the Tenable.io integration to fix an issue where the tenable-io-export-vulnerabilities command failed to execute.
```

Comment:
It is better to describe it this way: Fixed an issue with the -- argument of the -- command. 
Or: Added the option to send a comma-seperated list to the -- arguemt of the --command.